### ISSUE TITLE
Error processing InsertEnter Autocommands for "*":

### ISSUE THREAD

**Issue by gnmearacaun** [ChristianChiarulli/Neovim-from-scratch#236]
I'm getting this error persisting for each character I type or delete:
```
Error processing InsertEnter Autocommands for "*":
Error detected while processing InsertEnter Autocommands for "*":
E5108: Error executing lua /usr/local/share/nvim/runtime/lua/vim/lsp.lua:1513:
attempt to call field 'is_closing' (a nil value)
stack traceback:
        /usr/local/share/nvim/runtime/lua/vim/lsp.lua:1513: in function 'is_sto
pped'
        ...ck/packer/start/cmp-nvim-lsp/lua/cmp_nvim_lsp/source.lua:16: in func
tion 'is_available'
        ...pack/packer/start/cmp-nvim-lsp/lua/cmp_nvim_lsp/init.lua:57: in func
tion '_on_insert_enter'
        [string ":lua"]:1: in main chunk
```
so far I've just tried wiping my .cache/nvim folder and removing and reinstalling `cmp_nvim_lsp` and cmp_nvim_lua`

**Comment by emko** [comment]
yes same, was trying to learn how to setup the LSP from this repo but even when cloning and using the full config i get this error as well

**Comment by gnmearacaun** [comment]
~This is fixed in master~

**Comment by gnmearacaun** [comment]
Chris mentioned with reference to this: likely needs a new version of lspconfig, or completely switch to mason

**Comment by nairobi02** [comment]
> ~This is fixed in master~

I dont think i understand. Did you find a solution for this?

**Comment by gnmearacaun** [comment]
Fixed on master by #241
